website/content/docs/v0.7/Guides/upgrading-kubernetes.md

```diff
@@ -251,13 +251,13 @@ spec:
 ### Kubelet
 
 The Talos team now maintains an image for the `kubelet` that should be used starting with Kubernetes 1.19.
-The image for this release is `docker.io/autonomy/kubelet:v1.19.0`.
+The image for this release is `ghcr.io/talos-systems/kubelet:v1.19.0`.
```

Comment:
I think we shouldn't change that (or we should update the tag to 1.19.3)

Reply:
will change to `1.19.3` ... goal is to get rid of `docker.io/autonomy` as much as possible in 0.7.